contracts/CollateralTracker.sol

```diff
@@ -1214,10 +1214,24 @@ contract CollateralTracker is ERC20Minimal, Multicall {
             // transfer delegatee balance to delegator
             _transferFrom(delegatee, delegator, delegateeBalance);
 
-            unchecked {
-                // mint shares to complete to requested amount
-                _mint(delegator, shares - delegateeBalance);
-            }
+            // this is paying out protocol loss, so correct for that in the amount of shares to be minted
+            // X: total assets in vault
+            // Y: total supply of shares
+            // Z: desired value (assets) of shares to be minted
+            // N: total shares corresponding to Z
+            // T: transferred shares from liquidatee which are a component of N but do not contribute toward protocol loss
+            // Z = N * X / (Y + N - T)
+            // Z * (Y + N - T) = N * X
+            // ZY + ZN - ZT = NX
+            // ZY - ZT = N(X - Z)
+            // N = (ZY - ZT) / (X - Z)
+            // N = Z(Y - T) / (X - Z)
+            // subtract delegatee balance from N since it was already transferred to the delegator
+            _mint(
+                delegator,
+                Math.mulDiv(assets, totalSupply - delegateeBalance, totalAssets() - assets) -
```

Comment:
Ok, I had to check. The expression on the liquidation branch is different, but it turns out it's the exact same value:

From the [liquidation branch](https://github.com/panoptic-labs/panoptic-v1-core/blob/716158da90108f83a244e9afa1be60f1db379155/contracts/CollateralTracker.sol#L1122-L1143): 
```
            // must mint the correct amount of shares (∆) so that the redeemable value is exactly equal to A, the requested assets
            // tA: total assets in vault (ie. totalAssets())
            // tS: total supply of shares (ie. totalSupply)
            // ∆: desired number of shares to be minted
            // A: requested assets
            // B: transferred shares from liquidatee (ie. delegateeBalance)
            //
            // Redeemable value = assets === delegatorShares * totalAssets() / newTotalSupply
            //
            // A = (B + ∆) * tA / (tS + ∆)
            // ∆ * (tA - A) = A * tS - B * tA
            // ∆ = (A*tS - B*tA)/(tA-A)

            uint256 tA = totalAssets();
            unchecked {
                // mint shares to complete to requested amount
                _mint(
                    delegator,
                    Math.mulDiv(assets, totalSupply, tA - assets) -
                        Math.mulDiv(delegateeBalance, tA, tA - assets)
                );
            }
```

So the final expressions are:

`Math.mulDiv(assets, totalSupply - delegateeBalance, totalAssets() - assets) -delegateeBalance)`

versus

`Math.mulDiv(assets, totalSupply, tA - assets) - Math.mulDiv(delegateeBalance, tA, tA - assets)`

It's the exact same final expression once we expand yours.

Reply:
Yep. I did it that way to save a mulDiv.